docs/RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,84 @@
+
+Using Release Please takes much of the manual work out of creating a release, requiring only an approval from a maintainer to approve the release.
+
+# Release Please Action
+
+## Usage
+
+We use the [Release Please](https://github.com/google-github-actions/release-please-action) GitHub Action, which you can find in `./github/workflows/release-please.yml`.
+
+Once a PR is merged, the Action will automatically run and update a Release PR. It will automatically create the changelog and version number of the release. If subsequent PRs are merged, the Release PR will update to reflect these new changes. Once ready, the release PR can be merged and the main page of the repository will be updated with the new release.
+
+## Security
+
+Due to the restrictions on this repository, and the inability to enable Actions to create pull requests, this workflow uses a personal access token.
+
+You can generate a personal access token by going to your [profile](https://github.com/settings/profile) > Developer settings > Personal access tokens.
+
+- Token name: `RELEASE_PLEASE_TOKEN`
+- Expiration: `90 days`
+- Resource Owner: `Azure-Samples`
+- Repository access: `Only select repositories` > `chat-with-your-data-solution-accelerator`
+
+The personal access token must be regenerated every 90 days. It must have the following permissions in order for the GitHub Action to be able to create a release pull request:
+- Actions: `Read and write`
+- Contents: `Read and write`
+- Merge queues: `Read and write`
+- Metadata: `Read-only`
+- Pull requests: `Read and write`
+- Workflows: `Read and write`
+
+Once the personal access token has been generated, it should be stored in the repository Settings, under Security > Secrets and Variables > Actions > RELEASE_PLEASE_TOKEN.
+
+# Conventional Commits
+
+This is a [specification](https://www.conventionalcommits.org/en/v1.0.0/) which is commonly used to help communicate the nature of the changes in a pull request or commit. As in our case, it makes it easier to use automated tooling based on those namings. For this repository, we ask the PRs should be prefixed with one of the following `types`:
+  * feat: A new feature
+  * fix: A bug fix
+  * docs: Documentation only changes
+  * style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+  * refactor: A code change that neither fixes a bug nor adds a feature
+  * perf: A code change that improves performance
+  * test: Adding missing tests or correcting existing tests
+  * build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+  * ci: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+  * chore: Other changes that don't modify src or test files
+  * revert: Reverts a previous commit
+  * !: A breaking change is indicated with a `!` after the listed prefixes above, e.g. `feat!`, `fix!`, `refactor!`, etc.
```

Comment:
thinking aloud : seems like too many things to keep in mind while raising a PR. Is there a way to automate this ?

Reply:
I think I can add something back to the PR template so it is less things to remember